website/content/docs/auth/jwt/oidc_providers.mdx

```diff
@@ -295,7 +295,8 @@ Configuration steps:
 1. Find the issuer URL of the cluster.
 
    ```bash
-   timeout 30s kubectl proxy &
+   # When you're done, kill this background process, e.g. using: kill %1
```

Comment:
👍

Reply:
Actually I think you can make that `kill %%` which will kill the last job started in the background.

Reply:
Oh good point. I finally actually looked up jobspecs and `kill %kubectl\ proxy` also works but I prefer `kill %%` for the simplicity.